microsite/data/plugins/dbt.yaml

```diff
@@ -0,0 +1,13 @@
+---
+title: DBT Docs
+author: IIBenII
+authorUrl: https://github.com/IIBenII
+category: Discovery
+description: View DBT models and tests documentation.
+documentation: https://github.com/IIBenII/backstage-plugin-dbt
+iconUrl: https://avatars.githubusercontent.com/u/18339788?s=200&v=4
```

Comment:
Can you confirm that we can use this icon? It looks to be from dbt itself so we just need to be sure

Reply:
Hi, I'm not sure about it, and documentation is confusing (https://www.getdbt.com/trademark-guidelines). By safety I removed the icon

Reply:
Thanks, sorry about that.